plexpy/webserve.py

```diff
@@ -1709,6 +1782,7 @@ def get_recently_added_json(self, count='0', **kwargs):
             logger.warn(u"Unable to retrieve data for get_recently_added_json.")
 
     @cherrypy.expose
+    @addtoapi('get_episode_list')
```

Comment:
`get_episode_list_json` actually doesn't do anything. It can be removed completely.